electrum/gui/qt/wizard/wizard.py

```diff
@@ -0,0 +1,287 @@
+        QDialog.__init__(self, None)
+        self.app = app
+        self.config = config
+
+        # compat
+        self.gui_thread = threading.current_thread()
+
+        self.setMinimumSize(600, 400)
+
+        self.title = QLabel()
+
+        self.main_widget = QStackedWidget(self)
+
+        self.back_button = QPushButton(_("Back"), self)
+        self.back_button.clicked.connect(self.on_back_button_clicked)
+        self.next_button = QPushButton(_("Next"), self)
+        self.next_button.clicked.connect(self.on_next_button_clicked)
+        self.next_button.setDefault(True)
+        self.requestPrev.connect(self.on_back_button_clicked)
+        self.requestNext.connect(self.on_next_button_clicked)
+        self.logo = QLabel()
+
+        please_wait_layout = QVBoxLayout()
+        please_wait_layout.addStretch(1)
+        self.please_wait_l = QLabel(_("Please wait..."))
+        self.please_wait_l.setAlignment(Qt.AlignCenter)
+        please_wait_layout.addWidget(self.please_wait_l)
+        please_wait_layout.addStretch(1)
+        self.please_wait = QWidget()
+        self.please_wait.setVisible(False)
+        self.please_wait.setLayout(please_wait_layout)
+
+        error_layout = QVBoxLayout()
+        error_layout.addStretch(1)
+        # error_l = QLabel(_("Error!"))
+        # error_l.setAlignment(Qt.AlignCenter)
+        # error_layout.addWidget(error_l)
+        self.error_msg = WWLabel()
+        self.error_msg.setAlignment(Qt.AlignCenter)
+        error_layout.addWidget(self.error_msg)
+        error_layout.addStretch(1)
+        self.error = QWidget()
+        self.error.setVisible(False)
+        self.error.setLayout(error_layout)
+
+        outer_vbox = QVBoxLayout(self)
+        inner_vbox = QVBoxLayout()
+        inner_vbox.addWidget(self.title)
+        inner_vbox.addWidget(self.main_widget)
+        inner_vbox.addWidget(self.please_wait)
+        inner_vbox.addWidget(self.error)
+
+        scroll_widget = QWidget()
+        scroll_widget.setLayout(inner_vbox)
+        scroll = QScrollArea()
+        scroll.setFocusPolicy(Qt.NoFocus)
+        scroll.setWidget(scroll_widget)
+        scroll.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+        scroll.setWidgetResizable(True)
+        icon_vbox = QVBoxLayout()
+        icon_vbox.addWidget(self.logo)
+        icon_vbox.addStretch(1)
+        hbox = QHBoxLayout()
+        hbox.addLayout(icon_vbox)
+        hbox.addSpacing(5)
+        hbox.addWidget(scroll)
+        hbox.setStretchFactor(scroll, 1)
+        outer_vbox.addLayout(hbox)
+        outer_vbox.addLayout(Buttons(self.back_button, self.next_button))
+
+        self.icon_filename = None
+        self.set_icon('electrum.png')
+
+        self.start_viewstate = start_viewstate
+
+        self.show()
+        self.raise_()
+
+        QTimer.singleShot(40, self.strt)
+
+        # TODO: re-test if needed on macOS
+        # self.refresh_gui()  # Need for QT on MacOSX.  Lame.
```

Comment:
@SomberNight I suspect this isn't needed anymore, can you check?

Reply:
I guess it is not needed anymore. Tested on macos 10.15 with pyqt 5.15.9.
Does not look needed on master either - it was added in 2016 when still using pyqt4, so a lot has changed since.